blog/2023-06-30-app-competition-results.md

```diff
@@ -0,0 +1,65 @@
+    
+The result was close, the first, second and third place each had 31, 26 and 24% of the votes respectively, once more illustrating how competitive the field was this year, and how high quality the apps were. In the end, the DHIS2 Analytics Messenger application from HISP Tanzania was chosen as the winner.
+    
+The entire poll result can be seen on the Community of Practice here: https://community.dhis2.org/t/voting-polls-for-the-5-finalists-of-the-dhis2-app-competition/53502
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/2XYclHVLKrI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+ 
+
+### The winning application
+    
+The winning application has the following description provided by the HISP Tanzania team:
+    
+*The DHIS2 Analytics Messenger is an application that aims to make DHIS2 analytics data readily accessible to users. The application leverages the power of popular social messaging platforms, such as WhatsApp and Telegram, to deliver DHIS2 analytics to users. Given the increasing popularity of social messaging applications in recent years, there has been a growing potential for sharing DHIS2 analytics data through these media.*
+    
+Eric Chingalo, the Developer of the application, has posted a longer description, a video demonstration, and a way to get in contact with them, also, on the Community of Practice. I highly recommend everyone to have a look at the application, and if you’re interested in using the application or helping develop extra features on top of it, we highly recommend reaching out to them: https://community.dhis2.org/t/dhis2-analytics-messenger/53477
+    
+### Other applications
+    
+But just because we’ve got a winner, we should not disqualify the other amazing submissions of the app competition. Not only were the top-5 amazing, there were runner-ups that we felt also deserved a spot to present. Some of those got a spotlight to present their app in another session during the conference. A video of the presentations can be found on our Youtube channel, and directly, in the video below:
+    
+<iframe width="560" height="315" src="https://www.youtube.com/embed/7dlTw4loy7k" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+    
+One of the goals we want you to keep in mind, all of the applications you’ve seen in this article or on the CoP, are intended to be shared! So if you feel one of the applications can be suitable for your organisation as well, get in touch with the respective organisations and see how you can work together to improve your DHIS2 systems.
```

Comment:
I feel like dropping the `or on the CoP` from `all of the applications you’ve seen in this article or on the CoP` would be better as I guess people can sometime post about proprietary apps in the CoP that are not generally shareable?